### ISSUE TITLE
Use any arbitrary project as a terminology one

### ISSUE THREAD

**Issue by iafan** [translate/pootle#1720]
It would be good to have a '[ ] Terminology project' checkbox next to each project in administration panel, off by default for all projects except 'Terminology'. If I add my own project and flag it as a terminology one, the terminology will be gathered across all terminology projects.

Rationale:
- Users might want to turn the default terminology off.
- Users might have their own terminology projects that thay want to use in addition to or as a replacement to default Terminilogy.

**Comment by unho** [comment]
@iafan Is this still valid?

**Comment by iafan** [comment]
Yes, still valid, though the priority is low (because in our own deployment, we have one terminology for all translation projects)

**Comment by dwaynebailey** [comment]
Currently we can mark  a project as terminology by setting its checker style to 'terminology'.  But ideally it should actually not be a hack like this.
